omnigibson/simulator.py

```diff
@@ -105,8 +105,20 @@ def _launch_app():
         # sys.argv.append("--/log/outputStreamLevel=error")
         warnings.simplefilter("ignore", category=NumbaPerformanceWarning)
 
+    # Try to import the isaacsim module that only shows up when pip installing omniverse
+    try:
+        import isaacsim  # noqa: F401
+    except ImportError:
+        isaacsim = None
+
+    # Compute the Isaac Path next
+    if isaacsim is not None:
```

Comment:
nit: combine into single line? `isaac_path = os.environ["ISAAC_PATH"] if isaacsim is None else os.path.dirname(isaacsim.__file__)`

Reply:
Done